response.go

```diff
@@ -83,7 +83,9 @@ func (ctx *RequestCtx) FileResponse(fileName, filePath, mimeType string) error {
 
 	buff := bytebufferpool.Get()
 	buff.SetString("attachment; filename=")
-	buff.WriteString(fileName)
+	if _, err := buff.WriteString(fileName); err != nil {
```

Comment:
This err check is not necessary, because `buff.WriteString(fileName)`always returns a nil error.

Reply:
Got it. But the linter doesn't know it, so there are two ways to make it quiet

1. `_, _ = buff.WriteString`
2. `buff.WriteString() // nolint:errcheck`

What do you prefer?

Reply:
I prefer the second, the double `_` i don't like it at all. haha